frame/support/procedural/src/construct_runtime/mod.rs

```diff
@@ -184,6 +184,23 @@ fn construct_runtime_intermediary_expansion(
 	input: TokenStream2,
 	definition: ImplicitRuntimeDeclaration,
 ) -> Result<TokenStream2> {
+	let max_pallet_num = {
+		if cfg!(feature = "tuples-96") {
+			96
+		} else if cfg!(feature = "tuples-128") {
+			128
+		} else {
+			64
+		}
+	};
+
+	if definition.pallets.len() > max_pallet_num {
+		return Err(syn::Error::new(
+			input.span(),
+			"The number of pallets exceeds the maximum number of tuple elements.",
```

Comment:
Should include some information on how to fix this.

Reply:
yeah like an additional sentence like `format!("To increase this limit, enable the `tuples-{}` feature of [crate]", next_highest_max).as_str()`